packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/shipping/telephony-line-phone-accessories-shipping.controller.js

```diff
@@ -0,0 +1,88 @@
+import each from 'lodash/each';
+import filter from 'lodash/filter';
+
+import filterContact from './telephony-line-phone-accessories-shipping.service';
+
+export default class TelecomTelephonyLinePhoneAccessoriesShippingCtrl {
+  /* @ngInject */
+  constructor($q, TucTelephonyAccessoriesOrderProcess) {
+    this.$q = $q;
+    this.TucTelephonyAccessoriesOrderProcess = TucTelephonyAccessoriesOrderProcess;
+  }
+
+  $onInit() {
+    this.process = null;
+    this.loading = {
+      init: false,
+    };
+    this.shippingOptions = {
+      forceContactSelect: true,
+      payForRelay: true,
+    };
+    this.contactChoiceOptions = null;
+    this.contactDeferred = this.$q.defer();
+
+    this.init();
```

Comment:
Is the `init` function reused ? If not maybe the code coud be directly in this hook